### PR TITLE
Bigger size input

### LINKED ISSUE #127: The input field in the size is too small to see the value
screenshot 2021-05-11 14 40 27
As you can see in the image, even if you enter the height in the size input field of the component, you cannot see the value you entered because the width of the input field is very narrow.

### PR DESCRIPTION
**Pull request by mochi-sann**
fix #127 
The size input field was too small to input, so I made it bigger.

### CODE REVIEW

File: src/components/inspector/controls/TextControl.tsx [hunk 1]
```diff
@@ -33,7 +33,8 @@ const TextControl: React.FC<TextControlPropsType> = ({
         size="sm"
         value={value || ''}
         type="text"
-        width={hasColumn ? '3rem' : '100%'}
+        // width={hasColumn ? '10rem' : '100%'}
```
Comment (tlenclos) [inline src/components/inspector/controls/TextControl.tsx:36]: This should probably work while keeping this right ?

Comment (mochi-sann) [reply src/components/inspector/controls/TextControl.tsx:36]: I tried it and it worked correctly, I will change it.

Comment (mochi-sann) [reply src/components/inspector/controls/TextControl.tsx]: I fix it on https://github.com/premieroctet/openchakra/pull/131/commits/59f221bec1be0e7627a011fddd1380b591fe0823#diff-b9ac78e00ce182ec5001f330c2ae52942dd9c9e90dbb619125ada2384ed5cd73

**Comment by PierreCrb** [comment]
Hello ! This bug has been fixed in the PR https://github.com/premieroctet/openchakra/pull/139 ✅ 

Thank you !

**Pull request closed without merging**